README.md

```diff
@@ -1,92 +1,199 @@
 
-## Technical Preview Features 
- 
-Technical preview features are introduced to gain early feedback from developers. A preview feature is subject to change in the future releases. Using a preview feature in a production code base is therefore strongly discouraged.
-The only preview feature at the moment is `MultiNodeBatch` for K-Means, a stepless distributed algorithm based on oneCCL.
+<img style="display:inline;" height=auto width=auto src="https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/IDP%20scikit-learn%20accelearation%20compared%20with%20stock%20scikit-learn.png"></a>
 
-## oneDAL and Intel(R) DAAL
 
-Intel(R) oneAPI Data Analytics Library is an extenstion of Intel(R) Data Analytics Acceleration Library (Intel(R) DAAL). 
+The size of the data is growing exponentially, as is the need for high-performance and scalable frameworks to analyze all this data and extract some benefits from it.
+Besides superior performance on a single node, the distribution mechanics of DAAL provides excellent strong and weak scaling.
 
-This repository contains branches corresponding to both oneAPI and classical versions of the library. We encourage you to use oneDAL located under the `master` branch.
 
-|Product|Latest release|Branch|Resources|
-|-------|--------------|------|:-------------:|
-|oneDAL       |2021.1-beta06|[master](https://github.com/oneapi-src/oneDAL)</br>[rls/onedal-beta06-rls](https://github.com/oneapi-src/oneDAL/tree/rls/onedal-beta06-rls)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/en-us/oneapi/onedal)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Documentation](http://oneapi-src.github.io/oneDAL/)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/en-us/articles/system-requirements-for-oneapi-data-analytics-library#)|
-|Intel(R) DAAL|2020 Gold|[rls/daal-2020-rls](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-rls)</br>[rls/daal-2020-mnt](https://github.com/oneapi-src/oneDAL/tree/rls/daal-2020-mnt) (contains ongoing fixes)|&nbsp;&nbsp;&nbsp;[Home page](https://software.intel.com/en-us/daal)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[Developer Guide](https://software.intel.com/en-us/daal-programming-guide)&nbsp;&nbsp;&nbsp;</br>&nbsp;&nbsp;&nbsp;[System Requirements](https://software.intel.com/en-us/articles/intel-data-analytics-acceleration-library-2020-system-requirements)|
+Intel&reg; DAAL K-means fit, strong scaling result | Intel&reg; DAAL K-means fit, weak scaling results
+:-------------------------:|:-------------------------:
+![](https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/Intel%20DAAL%20KMeans%20strong%20scaling.png)  |  ![](https://github.com/PivovarA/daal/raw/dev/apivovar-newReadme/.github/charts/intel%20DAAL%20KMeans%20weak%20scaling.png)
 
-## Installation
+## Python API
```

Comment:
Python code in one repo, Python API description & example in another confuses a bit. Probably, need to add some intro or something to avoid confusion.

Reply:
No answer yet.

Reply:
I added an explanation to the Python API paragraph: `Intel(R) oneDAL has a python API that is provided as a standalone python library called daal4py`